lerna.json

```diff
@@ -18,6 +18,8 @@
     "libraries/testbot",
     "libraries/botframework-streaming-extensions",
     "libraries/botbuilder-streaming-extensions",
+    "libraries/Adapter/wechat",
```

Comment:
should we follow dotnet convention for the path name, e.g. Adapter/Bot.Builder.Adapters.Wechat